include/leo_object_storage.hrl

```diff
@@ -202,9 +202,10 @@
 %% AVS-Related
 %%--------------------------------------------------------------------
 %% AVS version strings
--define(AVS_HEADER_VSN_2_2,  <<"LeoFS AVS-2.2">>). %% leofs v0.14 - v1.0.0-pre1
--define(AVS_HEADER_VSN_2_4,  <<"LeoFS AVS-2.4">>). %% leofs v1.0.0-pre1 - current ver
--define(AVS_HEADER_VSN_TOBE, ?AVS_HEADER_VSN_2_4).
+-define(AVS_HEADER_VSN_2_2,  <<"LeoFS AVS-2.2">>). %% leofs v0.14 - v0.16
+-define(AVS_HEADER_VSN_2_4,  <<"LeoFS AVS-2.4">>). %% leofs v1.0 - v1.4
+-define(AVS_HEADER_VSN_3_0,  <<"LeoFS AVS-3.0">>). %% leofs v2.0
```

Comment:
No longer needed as sse-c related field will be included in the custom metadata field?